tools/src/test/python/dlpx/virtualization/_internal/test_file_util.py

```diff
@@ -80,13 +81,13 @@ def test_get_src_dir_path_is_dir_fail(mock_existing_path,
     @mock.patch('os.path.isabs', return_value=False)
     @pytest.mark.parametrize(
         'plugin_config_file_path, src_dir_path',
-        [(os.path.join(os.getenv('HOME'), 'plugin/file_name'), '.'),
+        [('plugin/file_name', '.'),
          ('/mongo/file_name', '/src'), ('/plugin/mongo/file_name', '/plugin'),
          ('/plugin/file_name', '/plugin/src/../..')])
```

Comment:
Is there a reason we don't use `os.path.join` for these paths? Is it because we use `os.path.join` in line 90 instead?